driver-core/src/main/java/com/datastax/driver/core/Metadata.java

```diff
@@ -241,11 +241,16 @@ public static String quoteIfNecessary(String id) {
   /**
    * We don't need to escape an identifier if it matches non-quoted CQL3 ids ([a-z][a-z0-9_]*), and
    * if it's not a CQL reserved keyword.
+   *
+   * <p>When 'Migrating from compact storage' after DROP COMPACT STORAGE on the table, it can have a
+   * column with an empty name. (See @jira_ticket JAVA-2174 for the reference) For that case, we
```

Comment:
Sorry to be nit-picky but please remove the `@jira_ticket` tag, this is an internal tag that is only recognized in tests by some of our CI tools (I don't even remember which one now). In any case this isn't a valid javadoc tag so it shouldn't appear in production code.

Reply:
thx, I was also not sure that It can be in production code. Changed